src/main/java/frc/robot/systems/ElevatorFSMSystem.java

```diff
@@ -336,19 +363,41 @@ private void handleL4State(TeleopInput input) {
 	abstract class ElevatorCommand extends Command {
 		private double target;
 
-		ElevatorCommand() { }
-
 		@Override
-		public void execute() { }
+		public void execute() {
```

Comment:
Again, I personally think the auto commands should just be setting the FSM state, so we can reuse the existing update() functionality.